### PR TITLE
Fixing and Updating the entire UX/UI of app

### LINKED ISSUE #207: Improve UI, separate activity screens from settings and fix spelling/capitalization
This can only be a step to the final UI. Merging this as WIP.

1. Rename area to settings 
2. Separate the different settings more clearly and add an explanation what they stand for. You can refer to any popular app like Twitter, FB, Instagram. See how settings areas usually look like.
3. Explain what the area Focus/Relax/VJ/SERIAL stands for. What is the functionality?
4. Please adhere to standard capitalization in apps and change as following (Please take care of such small things yourself in future!) e.g.:
* Audio Feedback
* 24 Bit
* Advanced Mode
5. Simulation: This should be a separate screen with a button "Load Sample" instead "Load Resources from Phone"
6. Avoid all capitals: Focus/Relax/VJ/SERIAL

### PR DESCRIPTION
**Pull request by jddeep**
Fixes #207 

**Changes**: Fixing and Updating the entire UX/UI of app

**Screenshot/s for the changes**:
Webp net-resizeimage (1)
Webp net-resizeimage (2)
Webp net-resizeimage (3)

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing:**
[app-debug.zip](https://github.com/fossasia/neurolab-android/files/3208146/app-debug.zip)

### CODE REVIEW

**Comment by CloudyPadmal** [comment]
Hello @jddeep , always attach an APK file to test.

File: app/src/main/res/layout/content_main.xml [hunk 1]
```diff
@@ -76,62 +77,9 @@
...
-                    android:id="@+id/cb_advanced_mode"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:layout_marginTop="@dimen/layout_margin_medium"
-                    android:text="@string/mode_advanced"
-                    android:textAppearance="@style/Base.TextAppearance.AppCompat.Medium" />
-
-            </LinearLayout>
+            android:layout_height="match_parent"
+            android:layout_marginTop="8dp"
```
Comment (suresh-gandhi) [inline app/src/main/res/layout/content_main.xml]: use dimens file for indirect referencing. avoid hardcoding.

File: app/src/main/res/layout/content_main.xml [hunk 2]
```diff
@@ -235,6 +187,20 @@
                 app:layout_constraintStart_toStartOf="@+id/btn_focus"
                 app:layout_constraintTop_toTopOf="@+id/guideline" />
 
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="8dp"
+                android:layout_marginLeft="8dp"
```
Comment (suresh-gandhi) [inline app/src/main/res/layout/content_main.xml]: same here

File: app/src/main/res/layout/content_main.xml [hunk 3]
```diff
@@ -235,6 +187,20 @@
...
+                android:layout_marginTop="36dp"
```
Comment (suresh-gandhi) [inline app/src/main/res/layout/content_main.xml]: here as well

File: app/src/main/res/layout/content_configs.xml [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+            android:textAppearance="@style/Base.TextAppearance.AppCompat.Medium" />
+
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="@string/audio_feedback_summary" />
+
+        <View
+            android:layout_width="match_parent"
+            android:layout_height="1dp"
```
Comment (suresh-gandhi) [inline app/src/main/res/layout/content_configs.xml]: avoid hardcoding

**Review by suresh-gandhi — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/content_main.xml (hunk 1 above)
Comment (jddeep) [reply app/src/main/res/layout/content_main.xml]: > use dimens file for indirect referencing. avoid hardcoding.

Yeah just was waiting for a review on the design. Will do these asap.

**Comment by jddeep** [comment]
@suresh-gandhi Did the formatting changes. Thanks for the review.

**Review by suresh-gandhi — APPROVED** [review]

**Pull request merged**